knora/dsplib/schemas/ontology.json

```diff
@@ -5,20 +5,18 @@
     "description": "JSON schema for DSP ontologies",
     "definitions": {
         "langstring": {
-            "anyOf": [
-                {
-                    "type": "object",
-                    "patternProperties": {
-                        "^(en|de|fr|it)": {
-                            "type": "string"
-                        }
-                    },
-                    "additionalProperties": false
-                },
-                {
+            "type": "object",
+            "patternProperties": {
+                "^(en|de|fr|it)": {
                     "type": "string"
                 }
-            ]
+            },
+            "additionalProperties": false
+        },
+        "ncname": {
+            "type": "string",
+            "pattern": "^[a-zA-Z0-9][a-zA-Z0-9_-]*$",
```

Comment:
@irinaschubert According to https://www.w3.org/TR/xml11/#NT-NameStartChar, `NCName` is not allowed to start with a number (but with underscore). Would that be valid for the API?